### PR TITLE
Deprecated import model method and added export model method.

### PR DESCRIPTION
**Pull request by maheshakya**

### CODE REVIEW

File: components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java [hunk 1]
```diff
@@ -241,9 +241,45 @@ public Response getModelSummary(@PathParam("modelId") long modelId) {
...
+        int tenantId = carbonContext.getTenantId();
+        String userName = carbonContext.getUsername();
+        try {
+            MLModelNew model = mlModelHandler.getModel(tenantId, userName, modelName);
+            if(model != null) {
+                final MLModel generatedModel = mlModelHandler.retrieveModel(model.getId());
+                StreamingOutput stream = new StreamingOutput() {
+                    public void write(OutputStream outputStream) throws IOException {
+                        ObjectOutputStream out = new ObjectOutputStream(outputStream);
+                        out.writeObject(generatedModel);
```
Comment (SupunS) [inline components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java:265]: Always close in/out streams in finally block.

File: components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java [hunk 2]
```diff
@@ -241,9 +241,45 @@ public Response getModelSummary(@PathParam("modelId") long modelId) {
...
+                    }
+                };
+                return Response.ok(stream).build();
+            } else {
+                return Response.status(Response.Status.NOT_FOUND).build();
+            }
+        } catch (Exception e) {
+            logger.error(String.format(
```
Comment (manoramahp) [inline components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java:273]: Catch the specific exception and handle it here.

File: components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java (hunk 1 above)
Comment (maheshakya) [inline components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java:265]: This method doesn't exist now.

File: components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java (hunk 2 above)
Comment (maheshakya) [inline components/ml/org.wso2.carbon.ml.rest.api/src/main/java/org/wso2/carbon/ml/rest/api/ModelApiV10.java:273]: This method doesn't exist now.
Handled correctly in the new method.

**Pull request merged**